public/js/components/SplitBox.js

```diff
@@ -24,11 +24,26 @@ const SplitBox = React.createClass({
 
   displayName: "SplitBox",
 
-  onMove(x) {
-    const node = ReactDOM.findDOMNode(this);
-    const width = (this.props.rightFlex ?
+  calculateWidth(x) {
+    const node = this.refs.splitBox;
+    return (this.props.rightFlex ?
       (node.offsetLeft + node.offsetWidth) - x :
       x - node.offsetLeft);
+  },
+
+  getflexEl() {
+    const el = this.props.rightFlex ? "right" : "left";
+    return this.refs[el];
+  },
+
+  onMove(x) {
+    const width = this.calculateWidth(x);
+    this.getflexEl().style.width = `${width}px`;
+  },
+
+  onStop(x) {
+    const el = this.getflexEl();
+    const width = parseInt(el.style.width, 10);
```

Comment:
This reduces a lot of CPU issues, but on my computer we were not CPU bound, so not sure if i want to fix it. Also, touching the style property might be taboo